src/pages/NewChairConsole.js

```diff
@@ -85,7 +85,11 @@ const Records = (props) => {
   const createActionButtons = (electionInfo, index) => {
     const e = electionInfo.election;
     const dar = electionInfo.dar;
-    const currentUserId = Storage.getCurrentUser().dacUserId;
+    const dacId = electionInfo.dac.dacId;
+    const currentUser = Storage.getCurrentUser();
+    const chairDacIds = map((role) => {return role.dacId;})(filter({roleId: 2})(currentUser.roles));
+    const currentUserId = currentUser.dacUserId;
+    const isChair = chairDacIds.includes(dacId);
```

Comment:
nit, naming. We know the user is a chair, so something like `isChairForDac` would be more intuitive.